controls/cis_rhel8.yml

```diff
@@ -0,0 +1,2300 @@
+      - l1_workstation
+    automated: yes
+    rules:
+      - sshd_disable_x11_forwarding
+
+  - id: 5.2.7
+    title: Ensure SSH MaxAuthTries is set to 4 or less (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - sshd_max_auth_tries_value=4
+      - sshd_set_max_auth_tries
+
+  - id: 5.2.8
+    title: Ensure SSH IgnoreRhosts is enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - sshd_disable_rhosts
+
+  - id: 5.2.9
+    title: Ensure SSH HostbasedAuthentication is disabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - disable_host_auth
+
+  - id: 5.2.10
+    title: Ensure SSH root login is disabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - sshd_disable_root_login
+
+  - id: 5.2.11
+    title: Ensure SSH PermitEmptyPasswords is disabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - sshd_disable_empty_passwords
+
+  - id: 5.2.12
+    title: Ensure SSH PermitUserEnvironment is disabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - sshd_do_not_permit_user_env
+
+  - id: 5.2.13
+    title: Ensure SSH Idle Timeout Interval is configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - sshd_idle_timeout_value=5_minutes
+      - sshd_set_idle_timeout
+      - sshd_set_keepalive_0
+      - var_sshd_set_keepalive=0
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5525
+  - id: 5.2.14
+    title: Ensure SSH LoginGraceTime is set to one minute or less (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 5.2.15
+    title: Ensure SSH warning banner is configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - sshd_enable_warning_banner
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5526
+  - id: 5.2.16
+    title: Ensure SSH PAM is enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 5.2.17
+    title: Ensure SSH AllowTcpForwarding is disabled (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - sshd_disable_tcp_forwarding
+
+  - id: 5.2.18
+    title: Ensure SSH MaxStartups is configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - sshd_set_maxstartups
+
+  - id: 5.2.19
+    title: Ensure SSH MaxSessions is set to 4 or less (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - sshd_set_max_sessions
+      - var_sshd_max_sessions=4
```

Comment:
I've changed this to your suggestion here so that we match the audit check for this rule but I'm _very_ confused where CIS got the title of this one from, since it does very clearly say `4`... 😕

Reply:
I filed an upstream issue asking about this one. https://workbench.cisecurity.org/community/14/tickets/13414

Reply:
Resolved using additional info from upstream and Draft Benchmark 1.1.0